modules/monitoring/manifests/checks/vpn_tunnels.pp

```diff
@@ -1,29 +0,0 @@
-# == Class: monitoring::checks::vpn_tunnels
-#
-# Sets up host checks for the gateways of remote VPN tunnels.
-#
-# === Parameters
-#
-# [*enabled*]
-#   Should we monitor the VPN tunnels?
-#
-# [*endpoints*]
-#   Hash of the endpoints to monitor
-#
-class monitoring::checks::vpn_tunnels (
-  $enabled = true,
```

Comment:
There's a bit of hieradata in the staging and production YAML files which could do with tidying up if this is being removed.